pkg/adaptor/cloud/libvirt/cloudinit.go

```diff
@@ -0,0 +1,51 @@
+// (C) Copyright Confidential Containers Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package libvirt
```

Comment:
Code wise, I have only one recommendation: we already have a common package for cloudinit (see `pkg/util/cloudinit`) , so what if move `createCloudInit()` and tests to that package?

Reply:
Hi @wainersm as far as I can tell the functionality in `pkg/util/cloudinit` deals with the creation of the `user-data` file while the function in the libvirt package is about the generation of the ISO file. 

I tend to not move that piece from libvirt, since that usecase to create the ISO in-memory and to assume an empty `vendor-data` is specific to the libvirt implementation.